packages/manager/apps/web/client/app/hosting/ftp/FTP.html

```diff
@@ -470,19 +470,19 @@
                                 <tr>
                                     <th
                                         scope="col"
-                                        data-translate="hosting_tab_FTP_multiftp_table_header_ftp_login"
```

Comment:
**question:** Does following translation keys could be removed from translation file then? (only for `Messages_fr_FR.json`)

- hosting_tab_FTP_multiftp_table_header_ftp_login
- hosting_tab_FTP_multiftp_table_header_ssh_login
- hosting_tab_FTP_multiftp_table_header_ftp_state

Reply:
Hello,

thanks for the review hosting_tab_FTP_multiftp_table_header_ftp_login and hosting_tab_FTP_multiftp_table_header_ssh_login can be removed.
hosting_tab_FTP_multiftp_table_header_ftp_state is still used.